microsoft-365/security/office-365-security/atp-safe-links-for-Teams.md

```diff
@@ -22,13 +22,15 @@ description: "Teams will now have access to safe links at the time of your click
 > [!IMPORTANT]
 > This feature is in **Public Preview** for customers in the Microsoft Teams Technology Adoption Program (TAP) as of Feb 28, 2020. This note will be removed from the article when Safe Links for Teams is more widely available.
 
-Microsoft Teams, an Office 365 Cloud-based application for managing your work, already uses safe attachments (for Office 365), but it will now have access to safe links at the time of your click. Whether you're using chats 1-on-1 Chats, between Groups, or in Channels, and Tabs, if you have a subscription to Office 365 ATP, you will have the ability to enable and use this safety measure.
+Microsoft Teams, an Office 365 Cloud-based application for managing your work, already uses safe attachments (for Office 365), but it will now have access to safe links at the time of your click. Whether you're using chats 1-on-1 Chats, between Groups, or in Channels, and Tabs, if you have a subscription to Office 365 ATP, you will have the ability to enable and use this safety measure.To learn more about licensing requirements, see [Microsoft 365 Tenant-Level Services Licensing Guidance](https://docs.microsoft.com/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance).
```

Comment:
```suggestion
Microsoft Teams, an Office 365 Cloud-based application for managing your work, already uses safe attachments (for Office 365), but it will now have access to safe links at the time of your click. Whether you're using chats 1-on-1 Chats, between Groups, or in Channels, and Tabs, if you have a subscription to Office 365 ATP, you will have the ability to enable and use this safety measure. To learn more about licensing requirements, see [Microsoft 365 Tenant-Level Services Licensing Guidance](https://docs.microsoft.com/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance).
```

Reply:
Please insert a space as indicated.

Reply:
Microsoft Teams是用于管理工作的基于Office 365云的应用程序，已经使用了安全附件（适用于Office 365），但是现在单击时便可以访问安全链接。无论您是在组之间，还是在频道和选项卡中使用一对一聊天，如果您已订阅Office 365 ATP，则可以启用和使用此安全措施。许可要求，请参阅[ Microsoft 365租户级服务许可指南 ]（https://docs.microsoft.com/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guided）。

Reply:
https://github.com/Vikramju